.github/workflows/sandbox-rebuild.yml

```
@@ -0,0 +1,28 @@
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1.7.0
+        with:
+          role-to-assume: ${{ secrets.AWS_CODEPIPELINE_ROLE_ARN }}
+          role-session-name: GitHub_to_AWS_via_FederatedOIDC
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Start Pipeline Execution
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:1:130: Double quote to prevent globbing and word splitting

(shellcheck)